### PR TITLE
adding python example of torque controller

### PR DESCRIPTION
**Pull request by andermi**
This is the python analog of the cpp torque controller utilizing the interface from buoy_msgs

`colcon test` passes

```
Label Time Summary:
copyright     =   0.72 sec*proc (1 test)
cppcheck      =   0.20 sec*proc (1 test)
cpplint       =   0.22 sec*proc (1 test)
flake8        =   0.28 sec*proc (1 test)
lint_cmake    =   0.18 sec*proc (1 test)
linter        =   2.40 sec*proc (8 tests)
pep257        =   0.21 sec*proc (1 test)
uncrustify    =   0.20 sec*proc (1 test)
xmllint       =   0.38 sec*proc (1 test)
```

Additionally, tested with fake PCRecord (..., rpm=1000, scale=1.1, retract=0.5, ...) publisher and PCWindCurrCommand service server:
`publishing #1: buoy_msgs.msg.PCRecord(header=std_msgs.msg.Header(stamp=builtin_interfaces.msg.Time(sec=0, nanosec=0), frame_id=''), seq_num=0, rpm=1000.0, sd_rpm=0.0, voltage=0.0, draw_curr_limit=0.0, bcurrent=0.0, wcurrent=0.0, torque=0.0, diff_press=0.0, bias_current=0.0, loaddc=0.0, scale=1.1, retract=0.5, target_v=0.0, target_a=0.0, status=0, charge_curr_limit=0.0)`
  
```
[INFO] [torque_controller.py-1]: process started with pid [652887]
[torque_controller.py-1] [INFO] [1649969552.152091599] [pb_torque_controller]: Subscribing to PCRecord on '/power_data'
[torque_controller.py-1] [INFO] [1649969552.153709464] [pb_torque_controller]: PBTorqueControlPolicy:
[torque_controller.py-1] 	Torque_constant: 0.438
[torque_controller.py-1] 	N_Spec: [   0.  300.  600. 1000. 1700. 4400. 6790.]
[torque_controller.py-1] 	Torque_Spec: [ 0.   0.   0.8  2.9  5.6  9.8 16.6]
[torque_controller.py-1] 	I_Spec: [ 0.          0.          1.82648402  6.62100457 12.78538813 22.37442922
[torque_controller.py-1]  37.89954338]
[torque_controller.py-1] [INFO] [1649969552.612414094] [pb_torque_controller]: Command Successful
```
  
`[INFO] [1649969552.611631979] [wind_curr_service]: got winding current command -3.641552686691284`

expected value:
`-0.5*1.1*2.9/0.438 = -3.64`

### CODE REVIEW

File: scripts/torque_controller.py
Comment (chapulina) [inline scripts/torque_controller.py:19]: I think we need and `<exec_depend>python3-scipy</exec_depend>` on `packages.xml`

**Review by chapulina — COMMENTED** [review]
Runs for me with the added dependency.

One thing to consider for the future is to break these examples into separate packages. This way, a user interested in using python could copy just the `buoy_torque_control_py` package and modify it to their needs.

File: scripts/torque_controller.py
Comment (andermi) [reply scripts/torque_controller.py:19]: added exec_depend

re: breaking into separate packages, I agree. It will be much cleaner and follows convention of other packages in ROS e.g. demo_nodes_cpp / demo_nodes_py. Thoughts @hamilton8415 ?

Comment (chapulina) [reply scripts/torque_controller.py:19]: > added exec_depend

I don't see it committed 🧐 

> re: breaking into separate packages,

I don't think we need to block this PR on that, it can be addressed later if needed

**Review by chapulina — APPROVED** [review]
LGTM once the exec depend is added

File: scripts/torque_controller.py
Comment (andermi) [reply scripts/torque_controller.py:19]: odd... I see it in the andermi/python_example branch on github (62fc107), but the PR didn't pick it up and only shows 1 commit

Comment (andermi) [reply scripts/torque_controller.py:19]: I changed the base away and then back to main and now it found the commit. :man_shrugging: ... "have you tried turning it off and on again?"

**Pull request merged**
